components/templates/events/eventDetails.tsx

```diff
@@ -0,0 +1,52 @@
+import Camera from 'components/icons/Camera.svg';
+import Star from 'components/icons/Star.svg';
+interface Event {
+	name?: string;
+	day: string;
+	createdAt: string;
+	stuffs?: [
+		{
+			id: string;
+			type: string;
+			name: string;
+		}
+	];
+}
```

Comment:
You are creating types by hand, when you can create them based on generated code from codegen, here is an example:

`type Event = Pick<GetAllEventsQuery["events"][number], "name" |  "day"  | "stuffs"> 
`

This way when schema change you will have you types correct

Reply:
Done